gen/plugin_test.go

```diff
@@ -171,9 +195,14 @@ func TestAddRootService(t *testing.T) {
 		g := newGenerateServiceBuilder(importer)
```

Comment:
this is some old code; while we're here, would you mind making use of subtests here with t.Run?

```
for _, tt := range tests {
  t.Run(tt.desc, func(t *testing.T) {
    // ...
  })
}
```

Reply:
Done